cmd/antrea-controller/controller.go

```diff
@@ -60,20 +61,26 @@ func run(o *Options) error {
 		return fmt.Errorf("error creating K8s clients: %v", err)
 	}
 	informerFactory := informers.NewSharedInformerFactory(client, informerDefaultResync)
+	crdInformerFactory := crdinformers.NewSharedInformerFactory(crdClient, informerDefaultResync)
```

Comment:
nit, consider grouping crdInformer entries together for easy read?

Reply:
Done